fuel/config_parser.py

```diff
@@ -96,6 +96,24 @@ def extra_downloader_converter(value):
     return value
 
 
+def multiple_paths_parser(value):
+    """Parses data_path argument.
+
+    Parameters
+    ----------
+    value : str
+        a string of data paths separated by  ":".
+
+    Returns
+    -------
+    value : list
+        a list of strings indicating each data paths.
+
+    """
+    value = value.split(os.path.pathsep)
```

Comment:
This will fail if `value` is already a list. To work around this, use

``` python
if isinstance(value, six.string_types):
    value = value.split(os.path.pathsep)
```

Incidentally, it's what makes the tests fail, I think.